_projects/2024/100452420/100452420.Rmd

```diff
@@ -0,0 +1,364 @@
+---
+title: "Serena Williams Career"
+description: |
+  it shows the ranking of the famous tennis player.
+categories: "2024"
+author: Diego Fernández-Alvarez
+date:  "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center", fig.showtext = TRUE)
+```
+
+This post details my process of replicating a chart that visualizes Serena Williams's world tennis ranking from 1998 to 2022. The original chart was published by The New York Times, and you can view it [here.](https://www.nytimes.com/2022/09/08/learning/whats-going-on-in-this-graph-sept-14-2022.html)
+
+
+Below is the original chart for reference:
+
+
+![Original Plot](../100452420/original_plot.png)
+
+## Original Chart
+
+The chart displays Serena Williams's fluctuating WTA (Women's Tennis Association) rankings throughout her career. 
+there are some relevant features such as a reverse y-axis that ranks her at the top with "No. 1" and descends to "20.". Also, the color orange in the line indicates periods when she held the No. 1 ranking. Finally there are some text annotations summarizing key moments in her career, such as her initial rise to No. 1, her return after childbirth, and her record-breaking 186 consecutive weeks at No. 1.
+
+I selected this graph for its explanatory power and simplicity, making it easy to interpret even for readers unfamiliar with tennis. Moreover, as a lifelong tennis fan, I have a personal connection to the topic.
+
+## Replication of the graph
+
+### Libraries and Data Preparation
+
+For this project, I primarily used the tidyverse package in R for data manipulation and visualization. Obtaining the dataset was challenging, as the official WTA website does not provide comprehensive rankings data. Fortunately, I discovered a GitHub repository by Jeff Sackmann that contained historical WTA rankings. However, the data was divided by decades (90s, 00s, 10s, and 20s) and included all players.
+
+To prepare the data, I joined the four tables into a single dataset (wta_data). After identifying Serena Williams's player ID (200033), I filtered the data to create the final dataset (serena_data) for the analysis.
+
+```{r} 
+library(tidyverse)
+
+# Reading the data
+wta90s <- as_tibble(read.csv("C:/Users/diegu/Downloads/wta_rankings_90s.csv"))
+wta00s <- as_tibble(read.csv("C:/Users/diegu/Downloads/wta_rankings_00s.csv"))
+wta10s <- as_tibble(read.csv("C:/Users/diegu/Downloads/wta_rankings_10s.csv"))
+wta20s <- as_tibble(read.csv("C:/Users/diegu/Downloads/wta_rankings_20s.csv"))
```

Comment:
Mmmh, there's an excel file but you invoke these CSVs. Then please ~~remove the excel file and~~ add these CSVs to the PR close to this Rmd. Then, call just `read.csv("wta_rankings_90s.csv")` etc. That is, in general, add all files you use to the PR and load them just with the relative path, removing any reference to paths in your on computer. ;-)